base/logger/R/logger.R

```diff
@@ -140,11 +140,12 @@ logger.severe <- function(msg, ..., wrap = TRUE) {
 ##' }
 logger.message <- function(level, msg, ..., wrap = TRUE) {
   if (logger.getLevelNumber(level) >= .utils.logger$level) {
-    dump.frames(dumpto = "dump.log")
-    calls <- names(dump.log)
+    call_dump <- NULL # to avoid "no visible binding" note from R check
+    utils::dump.frames(dumpto = "call_dump")
+    calls <- names(call_dump)
```

Comment:
This doesn't work and I'd have known that if I'd run the package tests before submitting like I was supposed to! 🙈 

(apparently `dump.frames` binds both a local AND a global variable, so creating a local one produces weird masking effects! So since we only care here about the names and not the frame contents, instead I'm generating the names in the same way `dump.frames` does internally)

@robkooper please check this -- do you see any problems that I and testthat didn't think of?

Reply:
@infotroph what's the sys.calls( ) here ?

Reply:
@moki1202 It's a base R call that tells you the whole stack of functions currently being executed, i.e. it's how you find out whether you're calling `foo()` from inside `bar(baz(quux(foo())))` or just `quux(foo()))`. You can see `help("sys.calls")` for more details.